article/basic-theory.tex

```diff
@@ -1,16 +1,13 @@
 \section{Basic Theory}
-When dealing with systems with a lot of users and a lot of content it can be beneficial to try and help users find content that they would like.
-This is usually done through recommender systems.
+When dealing with systems with a lot of users and a lot of content it causes information overload.
+Information overload can be minimized through recommender systems.
 There are broadly speaking two approaches to recommender systems which are content-based filtering and collaborative filtering.
-Content-based filtering is done by recommending items that are similar to items that the user has liked before.
-This approach uses features that have to be hand engineered by categorizing the items.
-Collaborative filtering on the other hand uses both relationship between users and the inter dependencies among items to provide recommendations \cite{Matrix-factorization-techniques}. 
+Content-based filtering takes recommending items that are similar to items that the user has liked before into account.
+Collaborative filtering on the other hand uses both relationship between users and the inter dependencies among items to provide recommendations \cite{Matrix-factorization-techniques}.
 This means that collaborative filtering recommends an item to a user based on a similar user.
 
-The advantage with collaborative filtering is that the features are learned automatically and it does not rely on hand engineered features which makes it more scalable for larger domains with many users and items.
-These features are called latent features.
-In general, collaborative filtering is more accurate than content-based filtering \cite{Matrix-factorization-techniques}.
-But collaborative filtering suffers from the cold start problem which means that it does not deal well with new users that have not rated any items yet.
+Collaborative filtering does often only utilizes users and their rated item, but it can still recommend item that are difficult to profile using content-based filtering \cite{Matrix-factorization-techniques}.
```

Comment:
```suggestion
Collaborative filtering does often only utilize users and their rated item, but it can still recommend items that are difficult to profile using content-based filtering \cite{Matrix-factorization-techniques}.
```